apps/core/utils.py

```diff
@@ -16,3 +18,15 @@ def delete_image_if_exists(instance):
                 f"An error occurred while deleting a file: {image_url}.\n{exc}"
             )
             raise
+
+
+def validate_city(data: dict[str, str | None]):
+    new_city_name, new_city_country = data.get("name"), data.get("country")
+    if new_city_name and new_city_country:
+        try:
+            new_location = City.objects.get(name=new_city_name, country=new_city_country)
+            return new_location
+        except City.DoesNotExist as _:
```

Comment:
what if user just changed the city, without changing a country, will this request work?

Reply:
Yes, because `City.objects.get(name=new_city_name, country=new_city_country)` will not find predefined city from db (fixtures).

Reply:
I'm talking more about line 25:`if new_city_name and new_city_country:`
if `data` will not have country, then we won't even try to find city objects
Another case if country is always in data, but if it's the case, then this function should accept both arguments city and country